src/Modal/index.jsx

```diff
@@ -183,6 +185,7 @@ class Modal extends React.Component {
               [styles['d-block']]: open,
               [styles.show]: open,
               [styles.fade]: !open,
+              [styles['is-ie']]: isIE11,
```

Comment:
should this really be `['is-ie-11']`? `isIE11` will be `false` for `Edge`, for example.